infra.ci.jenkins.io.tf

```diff
@@ -128,6 +128,36 @@ output "infraci_docs_jenkins_io_fileshare_serviceprincipal_writer_sp_password" {
   value     = module.infraci_docs_jenkins_io_fileshare_serviceprincipal_writer.fileshare_serviceprincipal_writer_sp_password
 }
 
+# Required to allow azcopy sync of stats.jenkins.io File Share
+module "infraci_stats_jenkins_io_fileshare_serviceprincipal_writer" {
+  source = "./.shared-tools/terraform/modules/azure-jenkinsinfra-fileshare-serviceprincipal-writer"
+
+  service_fqdn                   = "infra-ci-jenkins-io-fileshare_serviceprincipal_writer"
+  active_directory_owners        = [data.azuread_service_principal.terraform_production.id]
+  active_directory_url           = "https://github.com/jenkins-infra/azure"
+  service_principal_end_date     = "2024-09-19T23:00:00Z"
```

Comment:
Don't forget to add an updatecli manifest to update this date  + a team calendar event. Not blocking for this PR, but needed once merged.

Reply:
Calendar event already added to the Jenkins Infra team calendar.

Will create a PR for the updatecli manifest.

Reply:
https://github.com/jenkins-infra/azure/pull/737